open_spiel/python/examples/benchmark_games.py

```diff
@@ -57,8 +59,15 @@ def _rollout_until_timeout(game_name, time_limit, give_up_after):
       if len(state.history()) > give_up_after:
         num_giveups += 1
         break
-      action = random.choice(state.legal_actions(state.current_player()))
-      state.apply_action(action)
+      if not state.is_simultaneous_node():
```

Comment:
A simpler way to handle this is to load the game as turn based if it is simultaneous. Note that most algorithm (DQN, A2C) deal with simultaneous move games this way.

Reply:
I know. Should I add a flag (`convert_simultaneous_game_to_turn_based`) to do that?

As this module is done for benchmarking, I think it is still useful to benchmark simultaneous games (as the benchmark might help finding where the code can be optimized)

Reply:
I'd load it as turn based if it is simultaneous. But as if you prefer to handle the simultaneous case, it seems fine by me (@lanctot do you agree?).

Reply:
I'm fine either way, I see the benefit of both.

Reply:
Done: added a flag.